.vscode/settings.json

```diff
@@ -0,0 +1,14 @@
+{
+    // The number of spaces a tab is equal to. This setting is overridden
+    // based on the file contents when `editor.detectIndentation` is true.
+    "editor.tabSize": 4,
+
+    // Insert spaces when pressing Tab. This setting is overriden
+    // based on the file contents when `editor.detectIndentation` is true.
+    "editor.insertSpaces": true,
+
+    // When opening a file, `editor.tabSize` and `editor.insertSpaces`
+    // will be detected based on the file contents. Set to false to keep
+    // the values you've explicitly set, above.
+    "editor.detectIndentation": false
+}
```

Comment:
This will be vscode specific. An actual `.editorconfig` is editor agnostic. They should all (most?) respect it. And we're open sourcing this thing, so I think we should probably use `.editorconfig`

Ah. I see for VSCode, it's a plugin: https://marketplace.visualstudio.com/items?itemName=EditorConfig.EditorConfig